cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Babbage/LeadershipSchedule.hs

```diff
@@ -71,7 +71,7 @@ hprop_leadershipSchedule = H.integrationRetryWorkspace 2 "babbage-leadership-sch
       cTestnetOptions = cardanoDefaultTestnetOptions
                           { cardanoNodes = cardanoDefaultTestnetNodeOptions
                           , cardanoEpochLength = 1000
-                          , cardanoSlotLength = 0.02
+                          , cardanoSlotLength = 0.1
```

Comment:
Can you add an explanation in comments why this is `0.1` now?

Reply:
we were rewriting the shelley genesis file with 0.1 so it wasn't being used. I'll leave add a comment.

Reply:
Added a message in the commit